test/ClangImporter/Inputs/many-imports/module1.h

```diff
@@ -0,0 +1,2 @@
+// comment
+int x;
```

Comment:
Beware this test may fail on filesystems that collapse identical files, since clang will then see all of these modules as trying to use the same header.  We've been bitten by that with clang's module tests in the past.